docs/de/user/configuring_mobile.rst

```diff
@@ -0,0 +1,13 @@
+Konfiguration mobiler Apps für wallabag
+=======================================
+
+Schritte zum Konfigurieren der App
+----------------------------------
+
+- als erstes *neuen Client erstellen* in dem Abschnitt ``API Client Management`` wählen. Der Name deiner App ist nicht so wichtig wie die ``Client-ID`` und das ``Client-Secret``. Schreib diese zwei Werte auf.
```

Comment:
@Strubbl The first sentence doesn't look like a sentence, the rest is looking good 👍